app/assets/javascripts/application.js

```diff
@@ -4,6 +4,11 @@
 // It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
 // the compiled file.
 //
+//= require govuk_publishing_components/lib
+//= require govuk_publishing_components/components/button
+//= require govuk_publishing_components/components/feedback
+//= require govuk_publishing_components/components/govspeak
+//
 //= require jquery_ujs
 //= require jquery.ui.all
 //= require vendor/jquery/magna-charta.min
```

Comment:
I _think_ that the `govspeak` component already includes the Magna Charta library - let me check.

Reply:
[magna-charta](https://github.com/alphagov/govuk_publishing_components/blob/9531ad85ded5c74fa47b8d32c54a587edb39e44c/app/assets/javascripts/govuk_publishing_components/lib/govspeak/barchart-enhancement.js) is already in `govuk_publishing_components/lib` indeed

Reply:
Great spot, thanks, have removed!

Reply:
Oooh that brought the JS size down a little more, excellent. Have updated the PR description.